katas/test/src/pokerhands/joaquin/CardTest.cs

```diff
@@ -0,0 +1,53 @@
+namespace Card.Test.Joaquin;
+
+using PokerHand.Joaquin;
+
+public class CardTest
+{
+    Deck Deck = new Deck();
+
+    char[] values = { 'A', '9', '4', 'J' };
+    char[] suits = { 'C', 'D', 'H', 'S' };
+    [Fact]
+    public void ItShouldReturnAValidSuit()
+    {
+        for (int i = 0; i < values.Length; i++)
```

Comment:
Try to avoid one single letter variable names, even for iterators. For example, `index` is more readable than `i`